<!DOCTYPE html>
<html lang="en">
<head>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
  <!-- prend tout lecran, zoom de base x1 -->
  <meta name="description" content="">
  <meta name="author" content="">
  <link rel="icon" href="../logo.jpg">
  <!-- ligne pour icon -->

  <title>The Charvin Lab - Publications</title>

  <!-- Bootstrap core CSS -->
  <link href="../Main/bootstrap.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="../Main/publications.css" rel="stylesheet">
  <link href="../Main/sticky-footer.css" rel="stylesheet">
  <link href="../Main/navbar.css" rel="stylesheet">
  <link href="../Main/main.css" rel="stylesheet">
      <link href="../Main/totop.css" rel="stylesheet">

<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
<link href="https://fonts.googleapis.com/css?family=Merriweather:400,900,900i" rel="stylesheet">

</head>





<body>
  <!-- NAVBAR -->
  <nav class="navbar navbar-expand-md navbar-dark fixed-top bg-dark">
    <a class="navbar-brand reduce_padding" href=../>
      <img src="../logo2.png" class="img-fluid"
      onmouseover="this.src='../logo1.png';" 
      onmouseout="this.src='../logo2.png';"alt="Charvin Lab">
    </a> <!-- utilise JS -->

    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExampleDefault" aria-controls="navbarsExampleDefault" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarsExampleDefault">
      <ul class="navbar-nav mr-auto">
        <li class="nav-item">
          <a class="nav-link " href="../">Home <vr> </a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="../Research/Research.html">Research </a>
          </li>
          <li class="nav-item ">
            <a class="nav-link" href="../People/People.html">People </a>
          </li>
          <li class="nav-item active">
            <a class="nav-link" href="../Publications/Publications.html">Publications <span class="sr-only">(current)</span></a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="../Resources/Resources.html">Resources</a>
          </li>
          <li class="nav-item">
          <a class="nav-link" href="../Positions/Positions.html">Positions</a>
          </li>

          <li class="nav-item">
            <a class="nav-link" href="../Collaborators/Collaborators.html">Collaborations</a>
          </li>
          <li class="nav-item">
            <a class="nav-link" href="../Contact/Contact.html">Contact</a>
          </li>
        </ul>
      </div>
    </nav>
    <!-- END OF NAVBAR -->

    <main id="on_top">
     <button onclick="topFunction()" id="buttonscroll" title="Go to top"></button> 
      <div class="container" >
        <div class="row">
          <div class="col-md">
          <h4>Publications from the Lab  </h4>
        </div>
        </div>
        <hr>

<!-- Publi41 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-1">
            <h5 class="font-weight-bold">2020</h5>
          </div>
          <div class="col-md-2">
            <img src="41.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.biorxiv.org/content/10.1101/2020.11.25.395608v1">pH fluctuations drive waves of stereotypical cellular reorganizations during entry into quiescence</a>
            <p class="auteur">Jacquel Basile*, Aspert Théo*, Laporte Damien, Sagot Isabelle, Charvin Gilles</p>
            <p class="auteur">* Equally contributed to the work</p>
            <p class="cite"><strong>bioRxiv</strong>, (78)5:915-925, (2020) - <a class="doi" href="https://doi.org/10.1101/2020.11.25.395608 "> DOI: 10.1101/2020.11.25.395608 </a></p>

            <a class="small" data-toggle="collapse" href="#abstract41" role="button" aria-expanded="false" aria-controls="abstract41">
            Abstract            </a>

            <div class="collapse" id="abstract41">
              <p class="abstract text-justify">The life cycle of microorganisms is associated with dynamic metabolic transitions and complex cellular responses. In yeast, how metabolic signals control the progressive establishment of structural reorganizations observed in quiescent cells remains unclear. To address this question, we have developed a method that combines nutrient-limited proliferation assays at the population level with single-cell tracking to unravel the coordination between metabolic and structural transitions in cells during an unperturbed lifecycle. We show that non-monotonous internal pH fluctuations are in sync with successive waves of protein super-assemblies formation and ultimately lead to a cytosolic glass transition. Our results, therefore, suggest a simple model explaining how the complex developmental changes during the yeast life cycle are orchestrated by the sequence of metabolic transitions.</p>
            </div>
          </div>
        </div>

<!-- Publi40 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="40.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.cell.com/molecular-cell/pdf/S1097-2765(20)30258-6.pdf?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276520302586%3Fshowall%3Dtrue">Single-Cell Tracing Dissects Regulation of Maintenance and Inheritance of Transcriptional Reinduction Memory</a>
            <p class="auteur">Poonam Bheda, Diana Aguilar-Gómez, Nils B Becker, Johannes Becker, Emmanouil Stavrou, Igor Kukhtevich, Thomas Höfer, Sebastian Maerkl, Gilles Charvin, Carsten Marr, Antonis Kirmizis, Robert Schneider</p>
            <p class="cite"><strong>Mol. Cell</strong>, (78)5:915-925, (2020) - <a class="doi" href="https://www.cell.com/molecular-cell/pdf/S1097-2765(20)30258-6.pdf?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS1097276520302586%3Fshowall%3Dtrue"> DOI: 10.1016/j.molcel.2020.04.016 </a></p>

            <a class="small" data-toggle="collapse" href="#abstract40" role="button" aria-expanded="false" aria-controls="abstract40">
            Abstract            </a>

            <div class="collapse" id="abstract40">
              <p class="abstract text-justify">Transcriptional memory of gene expression enables adaptation to repeated stimuli across many organisms. However, the regulation and heritability of transcriptional memory in single cells and through divisions remains poorly understood. Here, we combined microfluidics with single-cell live imaging to monitor Saccharomyces cerevisiae galactokinase 1 ( GAL1) expression over multiple generations. By applying pedigree analysis, we dissected and quantified the maintenance and inheritance of transcriptional reinduction memory in individual cells through multiple divisions. We systematically screened for loss- and gain-of-memory knockouts to identify memory regulators in thousands of single cells. We identified new loss-of-memory mutants, which affect memory inheritance into progeny. We also unveiled a gain-of-memory mutant, elp6Δ, and suggest that this new phenotype can be mediated through decreased histone occupancy at the GAL1 promoter. Our work uncovers principles of maintenance and inheritance of gene expression states and their regulators at the single-cell level.</p>
            </div>
          </div>
        </div>


<!-- Publi39 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-1">
            <h5 class="font-weight-bold">2019</h5>
          </div>
          <div class="col-md-2">
            <img src="39.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://elifesciences.org/articles/48240">Proteostasis collapse halts G1 progression and delimits replicative lifespan</a>
            <p class="auteur">Moreno D F, Jenkins K, Morlot S, Charvin G, Csikász-Nagy A, Aldea M</p>
            <p class="cite"><strong>eLife</strong>, 8:e48240, (2019) - <a class="doi" href="https://elifesciences.org/articles/48240"> DOI: 10.7554/eLife.48240</a></p>

            <a class="small" data-toggle="collapse" href="#abstract39" role="button" aria-expanded="false" aria-controls="abstract39">
            Abstract            </a>

            <div class="collapse" id="abstract39">
              <p class="abstract text-justify">Loss of proteostasis and cellular senescence are key hallmarks of aging, but direct cause-effect relationships are not well understood. We show that most yeast cells arrest in G1 before death with low nuclear levels of Cln3, a key G1 cyclin extremely sensitive to chaperone status. Chaperone availability is seriously compromised in aged cells, and the G1 arrest coincides with massive aggregation of a metastable chaperone-activity reporter. Moreover, G1-cyclin overexpression increases lifespan in a chaperone-dependent manner. As a key prediction of a model integrating autocatalytic protein aggregation and a minimal Start network, enforced protein aggregation causes a severe reduction in lifespan, an effect that is greatly alleviated by increased expression of specific chaperones or cyclin Cln3. Overall, our data show that proteostasis breakdown, by compromising chaperone activity and G1-cyclin function, causes an irreversible arrest in G1, configuring a molecular pathway postulating proteostasis decay as a key contributing effector of cell senescence.</p>
            </div>
          </div>
        </div>



<!-- Publi37 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="37.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.cell.com/cell-reports/fulltext/S2211-1247(19)30798-3#.XSTmC_VC9LE">Excessive rDNA transcription drives the disruption in nuclear homeostasis during entry into senescence in budding yeast </a>
            <p class="auteur">Morlot S, Song J, Léger I, Matifas A, Gadal O, Charvin G </p>
            <p class="cite"><strong>Cell Rep</strong>, (28)2:408-422, (2019) - <a class="doi" href="https://www.cell.com/cell-reports/fulltext/S2211-1247(19)30798-3#.XSTmC_VC9LE">DOI: 10.1016/j.celrep.2019.06.032</a></p>

            <a class="small" data-toggle="collapse" href="#abstract37" role="button" aria-expanded="false" aria-controls="abstract37">
            Abstract            </a>

            <div class="collapse" id="abstract37">
              <p class="abstract text-justify">Budding yeast cells undergo a limited number of divisions before they enter senescence and die. Despite recent mechanistic advances, whether and how molecular events are temporally and causally linked during the transition to senescence remain elusive. Here, using real-time observation of the accumulation of extrachromosomal rDNA circles (ERCs) in single cells, we provide evidence that ERCs build up rapidly with exponential kinetics well before any physiological decline. We then show that ERCs fuel a massive increase in ribosomal RNA (rRNA) levels in the nucleolus, which do not mature into functional ribosomes. This breakdown in nucleolar coordination is followed by a loss of nuclear homeostasis, thus defining a chronology of causally related events leading to cell death. A computational analysis supports a model in which a series of age-independent processes lead to an age-dependent increase in cell mortality, hence explaining the emergence of aging in budding yeast.</p>
            </div>
          </div>
        </div>


<!-- Publi38 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="38.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0220694">COSPLAY: An expandable toolbox for combinatorial and swift generation of expression plasmids in yeast </a>
            <p class="auteur">Goulev Y, Matifas A, Heyer V, Reina-San-Martin B, Charvin G</p>
            <p class="cite"><strong>PLoS ONE</strong>, 14(8): e0220694, (2019) - <a class="doi" href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0220694">DOI: 10.1371/journal.pone.0220694
</a></p>

            <a class="small" data-toggle="collapse" href="#abstract38" role="button" aria-expanded="false" aria-controls="abstract38">
            Abstract            </a>

            <div class="collapse" id="abstract38">
              <p class="abstract text-justify">A large number of genetic studies in yeast rely on the use of expression vectors. To facilitate the experimental approach of these studies, several collections of expression vectors have been generated (YXplac, pRS series, etc.). Subsequently, these collections have been expanded by adding more diversity to many of the plasmid features, including new selection markers and new promoter sequences. However, the ever growing number of plasmid features makes it unrealistic for research labs to maintain an up-to-date collection of plasmids. Here, we developed the COSPLAY toolbox: a Golden Gate approach based on the scheme of a simple modular plasmid that recapitulates and completes all the properties of the pRS plasmids. The COSPLAY toolbox contains a basal collection of individual functional modules. Moreover, we standardized a simple and rapid, software-assisted protocol which facilitates the addition of new personalized modules. Finally, our toolbox includes the possibility to select a genomic target location and to perform a single copy integration of the expression vector.</p>
            </div>
          </div>
        </div>


<!-- Publi36 -->
        <div class="row justify-content-end my-4">

          <div class="col-md-2">
            <img src="selflearning.jpg" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.mdpi.com/2072-666X/10/5/311">Self-Learning Microfluidic Platform for Single-Cell Imaging and Classification in Flow</a>
            <p class="auteur">Constantinou I#, Jendrusch M#, Aspert T, Görlitz F, Schulze A, Charvin G, Knop M.</p>
            <p class="cite"><strong>Micromachines</strong>, 10: 311, (2019) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/31075890">DOI: 10.3390/mi10050311</a></p>

            <a class="small" data-toggle="collapse" href="#abstract36" role="button" aria-expanded="false" aria-controls="abstract36">
            Abstract            </a>

            <div class="collapse" id="abstract36">
              <p class="abstract text-justify">Single-cell analysis commonly requires the confinement of cell suspensions in an analysis chamber or the precise positioning of single cells in small channels. Hydrodynamic flow focusing has been broadly utilized to achieve stream confinement in microchannels for such applications. As imaging flow cytometry gains popularity, the need for imaging-compatible microfluidic devices that allow for precise confinement of single cells in small volumes becomes increasingly important. At the same time, high-throughput single-cell imaging of cell populations produces vast amounts of complex data, which gives rise to the need for versatile algorithms for image analysis. In this work, we present a microfluidics-based platform for single-cell imaging in-flow and subsequent image analysis using variational autoencoders for unsupervised characterization of cellular mixtures. We use simple and robust Y-shaped microfluidic devices and demonstrate precise 3D particle confinement towards the microscope slide for high-resolution imaging. To demonstrate applicability, we use these devices to confine heterogeneous mixtures of yeast species, brightfield-image them in-flow and demonstrate fully unsupervised, as well as few-shot classification of single-cell images with 88% accuracy.</p>
            </div>
          </div>
        </div>

        <!-- Publi35 -->
        <div class="row my-4 justify-content-end">

          <div class="col-md-1">
            <h5 class="font-weight-bold">2018</h5>
          </div>


          <div class="col-md-2">
            <img src="35.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="http://genesdev.cshlp.org/content/32/23-24/1499.long">Adaptation to DNA damage checkpoint in senescent telomerase-negative cells promotes genome instability </a>
            <p class="auteur">Coutelier H#, Xu Z#, Morisse MC, Lhuillier-Akakpo M, Pelet S, Charvin G, Dubrana K, Teixeira MT. </p>
            <p class="cite"><strong>Genes Dev.</strong>, 1;32(23-24):1499-1513, (2018) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/30463903">DOI: 10.1101/gad.318485.118</a></p>

            <a class="small" data-toggle="collapse" href="#abstract35" role="button" aria-expanded="false" aria-controls="abstract35">
            Abstract            </a>

            <div class="collapse" id="abstract35">
              <p class="abstract text-justify">In cells lacking telomerase, telomeres gradually shorten during each cell division to reach a critically short length, permanently activate the DNA damage checkpoint, and trigger replicative senescence. The increase in genome instability that occurs as a consequence may contribute to the early steps of tumorigenesis. However, because of the low frequency of mutations and the heterogeneity of telomere-induced senescence, the timing and mechanisms of genome instability increase remain elusive. Here, to capture early mutation events during replicative senescence, we used a combined microfluidic-based approach and live-cell imaging in yeast. We analyzed DNA damage checkpoint activation in consecutive cell divisions of individual cell lineages in telomerase-negative yeast cells and observed that prolonged checkpoint arrests occurred frequently in telomerase-negative lineages. Cells relied on the adaptation to the DNA damage pathway to bypass the prolonged checkpoint arrests, allowing further cell divisions despite the presence of unrepaired DNA damage. We demonstrate that the adaptation pathway is a major contributor to the genome instability induced during replicative senescence. Therefore, adaptation plays a critical role in shaping the dynamics of genome instability during replicative senescence. </p>
            </div>
          </div>
        </div>



        <!-- Publi34 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="34.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://elifesciences.org/articles/34025">Multiple inputs ensure yeast cell size homeostasis during cell cycle progression  </a>
            <p class="auteur">Garmendia-Torres, C., Tassy, O., Matifas, A., Molina, N., Charvin, G.</p>
            <p class="cite"><strong>eLife</strong>, 7:e34025, (2018) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/29972352">DOI: 10.7554/eLife.34025</a></p>

            <a class="small" data-toggle="collapse" href="#abstract34" role="button" aria-expanded="false" aria-controls="abstract34">
            Abstract            </a>

            <div class="collapse" id="abstract34">
              <p class="abstract text-justify">Coordination of cell growth with division is essential for proper cell function. In budding yeast, although some molecular mechanisms responsible for cell size control during G1 have been elucidated, the mechanism by which cell size homeostasis is established remains to be discovered. Here, we developed a new technique based on quantification of histone levels to monitor cell cycle progression in individual cells with unprecedented accuracy. Our analysis establishes the existence of a mechanism controlling bud size in G2/M that prevents premature onset of anaphase, and controls the overall size variability. While most G1 mutants do not display impaired size homeostasis, mutants in which cyclin B-Cdk regulation is altered display large size variability. Our study thus demonstrates that size homeostasis is not controlled by a G1-specific mechanism alone but is likely to be an emergent property resulting from the integration of several mechanisms that coordinate cell and bud growth with division.</p>
            </div>
          </div>
        </div>



        <!-- Publi33 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="../Research/instru.jpg" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S0091679X18301018">Controllable stress patterns in microfluidic devices</a>
            <p class="auteur">Goulev, Y., Matifas, A. , Charvin, G.</p>
            <p class="cite"><strong>Methods Cell Biol.</strong>, 147:29-40, (2018) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/30165960">DOI: 10.1016/bs.mcb.2018.07.003</a></p>

            <a class="small" data-toggle="collapse" href="#abstract33" role="button" aria-expanded="false" aria-controls="abstract33">
            Abstract            </a>

            <div class="collapse" id="abstract33">
              <p class="abstract text-justify">The generation of complex temporal stress patterns may be instrumental to investigate the adaptive properties of individual cells submitted to environmental stress on physiological timescale. However, it is difficult to accurately control stress concentration over time in bulk experiments. Here, we describe a microfluidics-based protocol to induce tightly controllable H2O2 stress in budding yeast while constantly monitoring cell growth with single cell resolution over multi-generation timescale. Moreover, we describe a simple methodology to produce ramping H2O2 stress to investigate the homeostatic properties of the H2O2 scavenging system.</p>
            </div>
          </div>
        </div>



        <!-- Publi32 -->
        <div class="row my-4 justify-content-end">

          <div class="col-md-1">
            <h5 class="font-weight-bold">2017</h5>
          </div>

          <div class="col-md-2">
            <img src="32.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://elifesciences.org/articles/23971">Nonlinear feedback drives homeostatic plasticity in H2O2 stress response</a>
            <p class="auteur">Goulev, Y., Morlot, S., Matifas, A., Huang, B., Molin, M., Toledano, M. B., Charvin, G. </p>
            <p class="cite"><strong>eLife</strong>, 6:e23971, (2017) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/28418333">DOI: 10.7554/eLife.23971</a></p>

            <a class="small" data-toggle="collapse" href="#abstract32" role="button" aria-expanded="false" aria-controls="abstract32">
            Abstract            </a>

            <div class="collapse" id="abstract32">
              <p class="abstract text-justify">Homeostatic systems that rely on genetic regulatory networks are intrinsically limited by the transcriptional response time, which may restrict a cell's ability to adapt to unanticipated environmental challenges. To bypass this limitation, cells have evolved mechanisms whereby exposure to mild stress increases their resistance to subsequent threats. However, the mechanisms responsible for such adaptive homeostasis remain largely unknown. Here, we used live-cell imaging and microfluidics to investigate the adaptive response of budding yeast to temporally controlled H2O2 stress patterns. We demonstrate that acquisition of tolerance is a systems-level property resulting from nonlinearity of H2O2 scavenging by peroxiredoxins and our study reveals that this regulatory scheme induces a striking hormetic effect of extracellular H2O2 stress on replicative longevity. Our study thus provides a novel quantitative framework bridging the molecular architecture of a cellular homeostatic system to the emergence of nonintuitive adaptive properties.</p>
            </div>
          </div>
        </div>



        <!-- Publi31 -->
        <div class="row my-4 justify-content-end">
    
<div class="col-md-1">
            <h5 class="font-weight-bold">2016</h5>
          </div>

<div class="col-md-2">
            <img src="31.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S0006349516300352?via%3Dihub">Kinetics of Formation and Asymmetrical Distribution of Hsp104-Bound Protein Aggregates in Yeast</a>
            <p class="auteur">Paoletti C, Quintin S, Matifas A, Charvin G. </p>
            <p class="cite"><strong>Biophys J.</strong>, 110(7):1605-14, (2016) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/27074685">DOI: 10.1016/j.bpj.2016.02.034</a></p>

            <a class="small" data-toggle="collapse" href="#abstract31" role="button" aria-expanded="false" aria-controls="abstract31">
            Abstract            </a>

            <div class="collapse" id="abstract31">
              <p class="abstract text-justify">Budding yeast cells have a finite replicative life span; that is, a mother cell produces only a limited number of daughter cells before it slows division and dies. Despite the gradual aging of the mother cell, all daughters are born rejuvenated and enjoy a full replicative lifespan. It has been proposed that entry of mother cells into senescence is driven by the progressive accumulation and retention of damaged material, including protein aggregates. This additionally allows the daughter cells to be born damage free. However, the mechanism underlying such asymmetrical segregation of protein aggregates by mother and daughter cells remains controversial, in part because of the difficulties inherent in tracking the dynamics and fate of protein aggregates in vivo. To overcome such limitations, we have developed single-cell real-time imaging methodology to track the formation of heat-induced protein aggregates in otherwise unperturbed dividing cells. By combining the imaging data with a simple computational model of protein aggregation, we show that the establishment of asymmetrical partitioning of protein aggregates upon division is driven by the large bud-specific dilution rate associated with polarized growth and the absence of significant mother/bud exchange of protein aggregates during the budded phase of the cell cycle. To our knowledge, this study sheds new light on the mechanism of establishment of a segregation bias, which can be accounted for by simple physical arguments.</p>
            </div>
          </div>
        </div>



        <!-- Publi30 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-1">
            <h5 class="font-weight-bold">2015</h5>
          </div>

<div class="col-md-2">
            <img src="30.png" class="mh100">
          </div>


          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.nature.com/articles/ncomms8680">Two routes to senescence revealed by real-time analysis of telomerase-negative single lineages</a>
            <p class="auteur">Xu Z, Fallet E, Paoletti C, Fehrmann S, Charvin G, Teixeira MT</p>
            <p class="cite"><strong>Nature Comm.</strong>, 6:7680, (2015) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/26158780">DOI: 10.1038/ncomms8680</a></p>

            <a class="small" data-toggle="collapse" href="#abstract30" role="button" aria-expanded="false" aria-controls="abstract30">
            Abstract            </a>

            <div class="collapse" id="abstract30">
              <p class="abstract text-justify">In eukaryotes, telomeres cap chromosome ends to maintain genomic stability. Failure to maintain telomeres leads to their progressive erosion and eventually triggers replicative senescence, a pathway that protects against unrestricted cell proliferation. However, the mechanisms underlying the variability and dynamics of this pathway are still elusive. Here we use a microfluidics-based live-cell imaging assay to investigate replicative senescence in individual Saccharomyces cerevisiae cell lineages following telomerase inactivation. We characterize two mechanistically distinct routes to senescence. Most lineages undergo an abrupt and irreversible switch from a replicative to an arrested state, consistent with telomeres reaching a critically short length. In contrast, other lineages experience frequent and stochastic reversible arrests, consistent with the repair of accidental telomere damage by Pol32, a subunit of polymerase δ required for break-induced replication and for post-senescence survival. Thus, at the single-cell level, replicative senescence comprises both deterministic cell fates and chaotic cell division dynamics.</p>
            </div>
          </div>
        </div>



        <!-- Publi29 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="29.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S096098221500353X?via%3Dihub">Oscillatory Flow Modulates Mechanosensitive klf2a Expression through trpv4 and trpp2 during Heart Valve Development</a>
            <p class="auteur">Heckel E, Boselli F, Roth S, Krudewig A, Belting HG, Charvin G, Vermot J. </p>
            <p class="cite"><strong>Curr Biol.</strong>, 25(10):1354-61, (2015) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/25959969">DOI: 10.1016/j.cub.2015.03.038</a></p>

            <a class="small" data-toggle="collapse" href="#abstract29" role="button" aria-expanded="false" aria-controls="abstract29">
            Abstract            </a>

            <div class="collapse" id="abstract29">
              <p class="abstract text-justify">In vertebrates, heart pumping is required for cardiac morphogenesis and altering myocardial contractility leads to abnormal intracardiac flow forces and valve defects. Among the different mechanical cues generated in the developing heart, oscillatory flow has been proposed to be an essential factor in instructing endocardial cell fate toward valvulogenesis and leads to the expression of klf2a, a known atheroprotective transcription factor. To date, the mechanism by which flow forces are sensed by endocardial cells is not well understood. At the onset of valve formation, oscillatory flows alter the spectrum of the generated wall shear stress (WSS), a key mechanical input sensed by endothelial cells. Here, we establish that mechanosensitive channels are activated in response to oscillatory flow and directly affect valvulogenesis by modulating the endocardial cell response. By combining live imaging and mathematical modeling, we quantify the oscillatory content of the WSS during valve development and demonstrate it sets the endocardial cell response to flow. Furthermore, we show that an endocardial calcium response and the flow-responsive klf2a promoter are modulated by the oscillatory flow through Trpv4, a mechanosensitive ion channel specifically expressed in the endocardium during heart valve development. We made similar observations for Trpp2, a known Trpv4 partner, and show that both the absence of Trpv4 or Trpp2 leads to valve defects. This work identifies a major mechanotransduction pathway involved during valve formation in vertebrates.</p>
            </div>
          </div>
        </div>



        <!-- Publi28 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="28.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S0091679X15000072?via%3Dihub">A quantitative approach to study endothelial cilia bending stiffness during blood flow mechanodetection in vivo</a>
            <p class="auteur">Boselli F, Goetz JG, Charvin G, Vermot J. </p>
            <p class="cite"><strong>Methods Cell Biol.</strong>, 127:161-7, (2015) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/25837390">DOI: 10.1016/bs.mcb.2015.01.006</a></p>

            <a class="small" data-toggle="collapse" href="#abstract28" role="button" aria-expanded="false" aria-controls="abstract28">
            Abstract            </a>

            <div class="collapse" id="abstract28">
              <p class="abstract text-justify">Primary cilia are necessary for shear stress sensing in different developing organs such as the kidneys and blood vessels. In endothelial cells (ECs), primary cilia bend in response to blood flow forces and are necessary for flow sensing as well as for the control of angiogenesis. The different parameters guiding cilia bending reflect the forces generated at the surface of the ECs and the mechanical properties of the endothelial cilia. Here, we present an approach allowing the calculation of the bending rigidity of endothelial cilia based on live imaging. The method relies on segmentation and mathematical modeling to extract the critical parameters needed for the calculation.</p>
            </div>
          </div>
        </div>



        <!-- Publi27 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="27.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://link.springer.com/protocol/10.1007%2F978-1-4939-1878-2_13">In silico control of biomolecular processes</a>
            <p class="auteur">Uhlendorf J, Miermont A, Delaveau T, Charvin G, Fages F, Bottani S, Hersen P, Batt G. </p>
            <p class="cite"><strong>Methods Mol Biol.</strong>, 1244:277-85, (2015) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/25487102">DOI: 10.1007/978-1-4939-1878-2_13</a></p>

            <a class="small" data-toggle="collapse" href="#abstract27" role="button" aria-expanded="false" aria-controls="abstract27">
            Abstract            </a>

            <div class="collapse" id="abstract27">
              <p class="abstract text-justify">By implementing an external feedback loop one can tightly control the expression of a gene over many cell generations with quantitative accuracy. Controlling precisely the level of a protein of interest will be useful to probe quantitatively the dynamical properties of cellular processes and to drive complex, synthetically-engineered networks. In this chapter we describe a platform for real-time closed-loop control of gene expression in yeast that integrates microscopy for monitoring gene expression at the cell level, microfluidics to manipulate the cells environment, and original software for automated imaging, quantification, and model predictive control. By using an endogenous osmo-stress responsive promoter and playing with the osmolarity of the cells environment, we demonstrate that long-term control can indeed be achieved for both time-constant and time-varying target profiles, at the population level, and even at the single-cell level.</p>
            </div>
          </div>
        </div>



                <!-- Publi26 -->
        <div class="row my-4 justify-content-end">

<div class="col-md-1">
            <h5 class="font-weight-bold">2014</h5>
          </div>

          <div class="col-md-2">
            <img src="26.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S0092867414013026?via%3Dihub">A memory system of negative polarity cues prevents replicative aging  </a>
            <p class="auteur">Meitinger F, Khmelinskii A, Morlot S, Kurtulmus B, Palani S, Andres-Pons A, Hub B, Knop M, Charvin G, Pereira G </p>
            <p class="cite"><strong>Cell</strong>, 159(5):1056-69, (2014) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/25416945">DOI: 10.1016/j.cell.2014.10.014</a></p>

            <a class="small" data-toggle="collapse" href="#abstract26" role="button" aria-expanded="false" aria-controls="abstract26">
            Abstract            </a>

            <div class="collapse" id="abstract26">
              <p class="abstract text-justify">Cdc42 is a highly conserved master regulator of cell polarity. Here, we investigated the mechanism by which yeast cells never re-establish polarity at cortical sites (cytokinesis remnants [CRMs]) that have previously supported Cdc42-mediated growth as a paradigm to mechanistically understand how Cdc42-inhibitory polarity cues are established. We revealed a two-step mechanism of loading the Cdc42 antagonist Nba1 into CRMs to mark these compartments as refractory for a second round of Cdc42 activation. Our data indicate that Nba1 together with a cortically tethered adaptor protein confers memory of previous polarization events to translate this spatial legacy into a biochemical signal that ensures the local singularity of Cdc42 activation. "Memory loss" mutants that repeatedly use the same polarity site over multiple generations display nuclear segregation defects and a shorter lifespan. Our work thus established CRMs as negative polarity cues that prevent Cdc42 reactivation to sustain the fitness of replicating cells.</p>
            </div>
          </div>
        </div>


        <!-- Publi25 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="25.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S2211124714000667?via%3Dihub">Endothelial Cilia Mediate Low Flow Sensing during Zebrafish Vascular Development</a>
            <p class="auteur">Goetz JG, Steed E, Ferreira RR, Roth S, Ramspacher C, Boselli F, Charvin G, Liebling M, Wyart C, Schwab Y, Vermot J. </p>
            <p class="cite"><strong>Cell Rep</strong>, 6(5):799-808, (2014) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/24561257">DOI: 10.1016/j.celrep.2014.01.032</a></p>

            <a class="small" data-toggle="collapse" href="#abstract25" role="button" aria-expanded="false" aria-controls="abstract25">
            Abstract            </a>

            <div class="collapse" id="abstract25">
              <p class="abstract text-justify">The pattern of blood flow has long been thought to play a significant role in vascular morphogenesis, yet the flow-sensing mechanism that is involved at early embryonic stages, when flow forces are low, remains unclear. It has been proposed that endothelial cells use primary cilia to sense flow, but this has never been tested in vivo. Here we show, by noninvasive, high-resolution imaging of live zebrafish embryos, that endothelial cilia progressively deflect at the onset of blood flow and that the deflection angle correlates with calcium levels in endothelial cells. We demonstrate that alterations in shear stress, ciliogenesis, or expression of the calcium channel PKD2 impair the endothelial calcium level and both increase and perturb vascular morphogenesis. Altogether, these results demonstrate that endothelial cilia constitute a highly sensitive structure that permits the detection of low shear forces during vascular morphogenesis.</p>
            </div>
          </div>
        </div>


        <!-- Publi24 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-1">
            <h5 class="font-weight-bold">2013</h5>
          </div>

          <div class="col-md-2">
            <img src="24.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S2211124713006803">Aging yeast cells undergo a sharp entry into senescence unrelated to the loss of mitochondrial membrane potential</a>
            <p class="auteur">Fehrmann S, Paoletti C, Goulev Y, Ungureanu A, Aguilaniu H, Charvin G. </p>
            <p class="cite"><strong>Cell Rep</strong>, 5(6):1589-99, (2013) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/24332850">DOI: 10.1016/j.celrep.2013.11.013</a></p>

            <a class="small" data-toggle="collapse" href="#abstract24" role="button" aria-expanded="false" aria-controls="abstract24">
            Abstract            </a>

            <div class="collapse" id="abstract24">
              <p class="abstract text-justify">In budding yeast, a mother cell can produce a finite number of daughter cells before it stops dividing and dies. Such entry into senescence is thought to result from a progressive decline in physiological function, including a loss of mitochondrial membrane potential (ΔΨ). Here, we developed a microfluidic device to monitor the dynamics of cell division and ΔΨ in real time at single-cell resolution. We show that cells do not enter senescence gradually but rather undergo an abrupt transition to a slowly dividing state. Moreover, we demonstrate that the decline in ΔΨ, which is observed only in a fraction of cells, is not responsible for entry into senescence. Rather, the loss of ΔΨ is an age-independent and heritable process that leads to clonal senescence and is therefore incompatible with daughter cell rejuvenation. These results emphasize the importance of quantitative single-cell measurements to decipher the causes of cellular aging.</p>
            </div>
          </div>
        </div>



                <!-- Publi23 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="23.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="http://dev.biologists.org/content/140/21/4426">Pulse propagation by a capacitive mechanism drives embryonic blood flow</a>
            <p class="auteur">Anton H, Harlepp S, Ramspacher C, Wu D, Monduc F, Bhat S, Liebling M, Paoletti C, Charvin G, Freund JB, Vermot J. </p>
            <p class="cite"><strong>Development</strong>, 140(21):4426-34, (2013) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/24089470">DOI: 10.1242/dev.096768 </a></p>

            <a class="small" data-toggle="collapse" href="#abstract23" role="button" aria-expanded="false" aria-controls="abstract23">
            Abstract            </a>

            <div class="collapse" id="abstract23">
              <p class="abstract text-justify">Pulsatile flow is a universal feature of the blood circulatory system in vertebrates and can lead to diseases when abnormal. In the embryo, blood flow forces stimulate vessel remodeling and stem cell proliferation. At these early stages, when vessels lack muscle cells, the heart is valveless and the Reynolds number (Re) is low, few details are available regarding the mechanisms controlling pulses propagation in the developing vascular network. Making use of the recent advances in optical-tweezing flow probing approaches, fast imaging and elastic-network viscous flow modeling, we investigated the blood-flow mechanics in the zebrafish main artery and show how it modifies the heart pumping input to the network. The movement of blood cells in the embryonic artery suggests that elasticity of the network is an essential factor mediating the flow. Based on these observations, we propose a model for embryonic blood flow where arteries act like a capacitor in a way that reduces heart effort. These results demonstrate that biomechanics is key in controlling early flow propagation and argue that intravascular elasticity has a role in determining embryonic vascular function.</p>
            </div>
          </div>
        </div>



                        <!-- Publi22 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-2">
            <img src="22.png" class="mh100">
          </div>

          <div class="col-md-9 pl-3">
            <a class="titre" href="https://academic.oup.com/nar/article/41/8/4640/2409200">Comparison of DNA decatenation by Escherichia coli topoisomerase IV and topoisomerase III: implications for non-equilibrium topology simplification. </a>
            <p class="auteur">Seol Y, Hardin AH, Strub MP, Charvin G, Neuman KC. 
</p>
            <p class="cite"><strong>Nucleic Acids Res.</strong>, 41(8):4640-9, (2013) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/23460205">DOI: 10.1093/nar/gkt136</a></p>

            <a class="small" data-toggle="collapse" href="#abstract22" role="button" aria-expanded="false" aria-controls="abstract22">
            Abstract            </a>

            <div class="collapse" id="abstract22">
              <p class="abstract text-justify">Type II topoisomerases are essential enzymes that regulate DNA topology through a strand-passage mechanism. Some type II topoisomerases relax supercoils, unknot and decatenate DNA to below thermodynamic equilibrium. Several models of this non-equilibrium topology simplification phenomenon have been proposed. The kinetic proofreading (KPR) model postulates that strand passage requires a DNA-bound topoisomerase to collide twice in rapid succession with a second DNA segment, implying a quadratic relationship between DNA collision frequency and relaxation rate. To test this model, we used a single-molecule assay to measure the unlinking rate as a function of DNA collision frequency for Escherichia coli topoisomerase IV (topo IV) that displays efficient non-equilibrium topology simplification activity, and for E. coli topoisomerase III (topo III), a type IA topoisomerase that unlinks and unknots DNA to equilibrium levels. Contrary to the predictions of the KPR model, topo IV and topo III unlinking rates were linearly related to the DNA collision frequency. Furthermore, topo III exhibited decatenation activity comparable with that of topo IV, supporting proposed roles for topo III in DNA segregation. This study enables us to rule out the KPR model for non-equilibrium topology simplification. More generally, we establish an experimental approach to systematically control DNA collision frequency.</p>
            </div>
          </div>
        </div>



        <!-- Publi21 -->
        <div class="row my-4 justify-content-end">
<div class="col-md-1">
            <h5 class="font-weight-bold">2012</h5>
          </div>

<div class="col-md-2">
            <img src="21.png" class="mh100">
          </div>


          <div class="col-md-9 pl-3">
            <a class="titre" href="https://doi.org/10.1073/pnas.1206810109">Long-term model predictive control of gene expression at the population and single-cell levels</a>
            <p class="auteur">Uhlendorf J, Miermont A, Delaveau T, Charvin G, Fages F, Bottani S, Batt G, Hersen P.</p>
            <p class="cite"><strong>PNAS</strong>,  109(35):14271-6, (2012) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3435223/">DOI: 10.1073/pnas.1206810109</a></p>

            <a class="small" data-toggle="collapse" href="#abstract21" role="button" aria-expanded="false" aria-controls="abstract21">
            Abstract            </a>

            <div class="collapse" id="abstract21">
              <p class="abstract text-justify">Gene expression plays a central role in the orchestration of cellular processes. The use of inducible promoters to change the expression level of a gene from its physiological level has significantly contributed to the understanding of the functioning of regulatory networks. However, from a quantitative point of view, their use is limited to short-term, population-scale studies to average out cell-to-cell variability and gene expression noise and limit the nonpredictable effects of internal feedback loops that may antagonize the inducer action. Here, we show that, by implementing an external feedback loop, one can tightly control the expression of a gene over many cell generations with quantitative accuracy. To reach this goal, we developed a platform for real-time, closed-loop control of gene expression in yeast that integrates microscopy for monitoring gene expression at the cell level, microfluidics to manipulate the cells’ environment, and original software for automated imaging, quantification, and model predictive control. By using an endogenous osmostress responsive promoter and playing with the osmolarity of the cells environment, we show that long-term control can, indeed, be achieved for both time-constant and time-varying target profiles at the population and even the single-cell levels. Importantly, we provide evidence that real-time control can dynamically limit the effects of gene expression stochasticity. We anticipate that our method will be useful to quantitatively probe the dynamic properties of cellular processes and drive complex, synthetically engineered networks.</p>
            </div>
          </div>
        </div>



        <!-- Publi20 -->
        <div class="row my-4 justify-content-end">
          <div class="col-md-1">
            <h5 class="font-weight-bold">2011</h5>
          </div>

          <div class="col-md-2">
            <img src="20.png" class="mh100">
          </div>

          <div class="col-md-9 mb-3 pl-3">
            <a class="titre" href="https://www.sciencedirect.com/science/article/pii/S1097276511000438?via%3Dihub">Ultrasensitivity and positive feedback to promote sharp mitotic entry.</a>
            <p class="auteur">Goulev Y, Charvin G. </p>
            <p class="cite"><strong>Mol Cell.</strong>,  41(3):243-4, (2011) - <a class="doi" href="https://www.ncbi.nlm.nih.gov/pubmed/21292155">DOI: 10.1016/j.molcel.2011.01.016</a></p>

            <a class="small" data-toggle="collapse" href="#abstract20" role="button" aria-expanded="false" aria-controls="abstract20">
            Abstract            </a>

            <div class="collapse" id="abstract20">
              <p class="abstract text-justify">In this issue, Trunnell et al. (2011) show that in mitotic entry the positive feedback that drives the activation of cyclin-dependent kinase (Cdk) involves a very ultrasensitive step of phosphorylation of Cdc25C by Cdk, thus strongly contributing to the switch-like behavior of this essential cell-cycle transition.</p>
            </div>
          </div>
        </div>
</div>
    </main>


    <!-- FOOTER -->
              <footer class="footer container-fluid">
            <div class="row h-75" > 
              <div class="col-3 h-100">
               <a href="http://www.igbmc.fr"><img src="../logo_igbmc.png" class="img-fluid mx-auto" alt="logo_igbmc"></a>
             </div>
             <div class="col-3 h-100">
               <a href="http://www.unistra.fr"><img src="../logo_unistra.png" class="img-fluid mx-auto" alt="logo_igbmc"></a>
             </div>
             <div class="col-3 h-100">
               <a href="http://www.cnrs.fr"><img src="../logo_cnrs.png" class="img-fluid mx-auto" alt="logo_igbmc"></a>
             </div>
             <div class="col-3 h-100">
               <a href="http://www.inserm.fr"><img src="../logo_inserm.png" class="img-fluid mx-auto" alt="logo_igbmc"></a>
             </div>
           </div>
           <div class="row h-25"> 
            <div class="col-md-12 h-100">
              <hr>
              <p class="footer-text text-center"> © Théo ASPERT for Charvin Lab 2019 All rights reserved - Hosted by IGBMC - <a class="text-dark font-weight-bold" href="../Contact/Contact.html">Contact Us</a></p>
            </div>
          </div>

        </footer>
  <!-- END OF FOOTER -->
    <!-- Bootstrap core JavaScript
      ================================================== -->
      <!-- Placed at the end of the document so the pages load faster -->
      <script src="../Main/jquery-3.js"></script>
      <script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery-slim.min.js"><\/script>')</script>
      <script src="../Main/popper.js"></script>
      <script src="../Main/bootstrap.js"></script>
      <script src="../Main/totop.js"></script>

    </body>

    </html>

